### PR TITLE
Add support for nerdctl stop.

### PR DESCRIPTION
**Pull request by shishir-a412ed**
Signed-off-by: Shishir Mahajan <smahajan@roblox.com>

### CODE REVIEW

**Comment by shishir-a412ed** [comment]
```
root@vagrant:~/go/src/github.com/nerdctl# nerdctl ps
CONTAINER ID    IMAGE                              COMMAND        CREATED          STATUS     PORTS    NAMES
19e590475251    docker.io/library/ubuntu:latest    "sleep 300"    6 seconds ago    Running
74685a0b0f8a    docker.io/library/ubuntu:latest    "sleep 300"    4 seconds ago    Running
d60969311fa7    docker.io/library/ubuntu:latest    "sleep 300"    5 seconds ago    Running
root@vagrant:~/go/src/github.com/nerdctl# nerdctl stop $(nerdctl ps -q)
19e590475251
74685a0b0f8a
d60969311fa7
```

/cc @AkihiroSuda

File: stop.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+		}
+
+		status, err := task.Status(ctx)
+		if err != nil {
+			return err
+		}
+
+		if status.Status != containerd.Running {
+			// Task is not running anymore, no need to SIGKILL.
+			continue
```
Comment (AkihiroSuda) [inline stop.go:119]: Don't we need to print ID before continue?

**Comment by AkihiroSuda** [comment]
LGTM, but needs rebase, sorry

> root authored and shishir-a412ed 

Seems a configuration error

**Comment by shishir-a412ed** [comment]
Yeah I was just doing that :)

File: stop.go (hunk 1 above)
Comment (shishir-a412ed) [reply stop.go:120]: Fixed.

**Comment by shishir-a412ed** [comment]
@AkihiroSuda Rebased.

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**